lib/transactions.js

```diff
@@ -28,14 +28,10 @@ exports.createGeneration = (rpcData, blockReward, feeReward, recipients, poolAdd
 
     // Set sapling or overwinter to either true OR block height to activate.
     // NOTE: if both are set, sapling will be used.
-    if (coin.sapling) {
-        if (coin.sapling === true || (typeof coin.sapling === 'number' && coin.sapling <= blockHeight)) {
-            txb.setVersion(bitcoin.Transaction.ZCASH_SAPLING_VERSION);
-        }
-    } else if (coin.overwinter) {
-        if (coin.overwinter === true || (typeof coin.overwinter === 'number' && coin.overwinter <= blockHeight)) {
-            txb.setVersion(bitcoin.Transaction.ZCASH_OVERWINTER_VERSION);
-        }
+    if (coin.sapling === true || (typeof coin.sapling === 'number' && coin.sapling <= blockHeight)) {
```

Comment:
Does this mean you can enter a block height or true for sapling in coin.json?

Reply:
Never mind I see the change made. 👍

Reply:
To answer your question, yes -- you can set sapling to either true or the block height that sapling activates.